src/platform/user/profile/vet360/containers/ReceiveTextMessages.jsx

```diff
@@ -101,8 +101,8 @@ class ReceiveTextMessages extends React.Component {
             checked={!!this.props.profile.vet360.mobilePhone.isTextPermitted}
             label={
               <span>
-                Please check the box if you want to receive text message (SMS)
-                reminders for your VA health care appointments.
+                We’ll send VA health care appointment text reminders to this
```

Comment:
This message will now also be updated on Profile 1, but that should be OK per Tressa.